_episodes/042-ben-striegel.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "History of Rust with Ben Striegel"
+date: 2021-10-15T01:25:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e042-ben-striegel.mp3
+duration: "1:32:39"
+length: "89712021"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Ben Striegel](https://github.com/bstrie), a member of Rust's official community outreach team, about the history of Rust. 
+
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps 
+
+- [@1:56] - What got Ben first interested in Rust?
+- [@3:03] - How Ben got involved
+- [@9:36] - Rust 1.0
+- [@16:21] - What does [move](https://doc.rust-lang.org/std/keyword.move.html) mean?
+- [@17:36] - The [Borrow Checker](https://rustc-dev-guide.rust-lang.org/borrow_check.html)
+- [@20:04] - What language was the Rust compiler first written in?
+- [@25:04] - Choosing [LLVM](https://llvm.org/) over [GCC](https://gcc.gnu.org/)
+- [@33:28] - 2 ways to target Windows
+- [@34:39] - [libc](https://crates.io/crates/libc) and [musl](https://www.musl-libc.org/)
+- [@36:22] - Rust Editions
+- [@46:46] - Does Rust have a small standard library?
+- [@54:18] - Why [TOML](https://toml.io/en/)? TOML vs [YAML](https://yaml.org/)
+- [@58:53] - ["Tree shaking"](https://webpack.js.org/guides/tree-shaking/) in Rust?
+- [@01:00:48] - Who created Cargo?
+- [@01:02:26] - Rust's milestones
+- [@01:07:42] - [Mozilla 2020 layoffs](https://blog.mozilla.org/blog/2020/08/11/changing-world-changing-mozilla/)
+- [Discussion on /r/rust](https://www.reddit.com/r/rust/comments/i7stjy/how_do_mozilla_layoffs_affect_rust/)
```

Comment:
```suggestion
  - [Discussion on /r/rust](https://www.reddit.com/r/rust/comments/i7stjy/how_do_mozilla_layoffs_affect_rust/)
```
This should be indented since it's related to the item above.